source/PyMaterialX/CMakeLists.txt

```diff
@@ -56,6 +56,9 @@ endif()
 if (MATERIALX_BUILD_GEN_ARNOLD)
     add_subdirectory(PyMaterialXGenArnold)
 endif()
+if (MATERIALX_BUILD_GEN_MDL)
```

Comment:
I think this build flag has been removed and we are always building MDL now, same as with OSL and GLSL.